web/src/tippyjs.ts

```diff
@@ -520,4 +520,22 @@ export function initialize(): void {
         placement: "bottom",
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: "#move_topic_to_stream_widget_wrapper",
+        onShow(instance) {
+            if ($("#move_topic_to_stream_widget").prop("disabled")) {
+                instance.setContent(
+                    $t({
+                        defaultMessage:
+                            "You do not have permission to move messages to another stream in this organization.",
+                    }),
+                );
+                return undefined;
+            }
+            instance.destroy();
```

Comment:
Is this required? I think `return false` sufficed here to avoid leaking a tooltip; can you check the `tippy` documentation?

Reply:
Yeah, ```return false``` is sufficient. I have removed ```instance.destroy()```.